vector/src/main/res/values/styles_riot.xml

```diff
@@ -370,5 +370,103 @@
+    </style>
+
+    <style name="WidgetButtonSocialLogin.Google.Dark" >
+        <item name="android:backgroundTint">@color/button_social_google_background_selector_dark</item>
+        <item name="android:textColor">@color/white</item>
+    </style>
+
+    <style name="WidgetButtonSocialLogin.Github" parent="WidgetButtonSocialLogin">
+        <item name="icon">@drawable/ic_social_github</item>
+    </style>
+
+    <style name="WidgetButtonSocialLogin.Github.Light">
+        <item name="iconTint">@android:color/black</item>
+        <item name="android:textColor">@color/black</item>
+        <item name="android:backgroundTint">@color/white</item>
+    </style>
+
+    <style name="WidgetButtonSocialLogin.Github.Dark" >
+        <item name="iconTint">@android:color/white</item>
+        <item name="android:textColor">@color/white</item>
+        <item name="android:backgroundTint">@color/black</item>
+    </style>
+
+
+    <style name="WidgetButtonSocialLogin.Facebook" parent="WidgetButtonSocialLogin">
+        <item name="icon">@drawable/ic_social_facebook</item>
+    </style>
+
+    <style name="WidgetButtonSocialLogin.Facebook.Light">
+        <item name="strokeColor">#3877EA</item>
+        <item name="strokeWidth">1dp</item>
+        <item name="iconTint">#3877EA</item>
+        <item name="android:textColor">#3877EA</item>
+        <item name="android:backgroundTint">@color/white</item>
+    </style>
+
+    <style name="WidgetButtonSocialLogin.Facebook.Dark" >
+        <item name="iconTint">@android:color/white</item>
+        <item name="android:textColor">@color/white</item>
+        <item name="android:backgroundTint">#3877EA</item>
+    </style>
+
+
+    <style name="WidgetButtonSocialLogin.Twitter" parent="WidgetButtonSocialLogin">
+        <item name="icon">@drawable/ic_social_twitter</item>
+    </style>
+
+    <style name="WidgetButtonSocialLogin.Twitter.Light" >
+        <item name="iconTint">#5D9EC9</item>
+        <item name="android:textColor">#5D9EC9</item>
+        <item name="android:backgroundTint">@color/white</item>
+    </style>
+
+    <style name="WidgetButtonSocialLogin.Twitter.Dark">
+        <item name="iconTint">@color/white</item>
+        <item name="android:textColor">@color/white</item>
+        <item name="android:backgroundTint">#5D9EC9</item>
+    </style>
+
+
+    <style name="WidgetButtonSocialLogin.Apple" parent="WidgetButtonSocialLogin">
+        <item name="icon">@drawable/ic_social_apple</item>
+    </style>
+
+    <style name="WidgetButtonSocialLogin.Apple.Light" >
+        <item name="iconTint">@color/white</item>
+        <item name="android:textColor">@color/white</item>
+        <item name="android:backgroundTint">@color/black</item>
+    </style>
+
+    <style name="WidgetButtonSocialLogin.Apple.Dark">
+        <item name="iconTint">@color/black</item>
+        <item name="android:textColor">@color/black</item>
+        <item name="android:backgroundTint">@color/white</item>
+    </style>
```

Comment:
Dark and light variants should be moved to new dedicated files for light and dark styles, which will allow comparison of those files.

Reply:
I moved the new styles in a dedicated file